install/installation-windows.md

```diff
@@ -46,6 +46,8 @@ current PostgreSQL installation, do not install TimescaleDB using this method.
     click `Edit...`. In the `Edit environment variable` dialog, click `New` and
     type the path to your PostgreSQL `pg_config` file. It should
     be `C:\Program Files\PostgreSQL\14\bin\`. Click `OK` to save your changes.
+1.  If installing PostgreSQL version 15.1.1 or above, make sure OpenSSL 1.1.1 is
+    installed on the system.
```

Comment:
This is actually a prerequisite for the procedure, not a step. The test is that a step always starts with a verb (like "Select the option") or a location (like "In the dialog, select...").  Pre-reqs are listed above the procedure with a stem sentence like "Before you start, make sure you ...". You can see an example here: https://docs.timescale.com/install/latest/self-hosted/installation-source/

Reply:
Also, more about writing processes and procedures here: https://docs.timescale.com/timescaledb/latest/contribute-to-docs/#processes-and-procedures